### PR TITLE
Update path.ex

### PR DESCRIPTION
**Pull request by princemaple**
fix invalid path

### CODE REVIEW

File: lib/elixir/lib/path.ex [hunk 1]
```diff
@@ -207,7 +207,7 @@ defmodule Path do
 
       Path.type("D:/usr/local/bin") #=> :absolute
       Path.type("usr/local/bin")    #=> :relative
-      Path.type("D:bar.ex")         #=> :volumerelative
+      Path.type("D:/bar.ex")         #=> :volumerelative
```
Comment (michallepicki) [inline lib/elixir/lib/path.ex:210]:
```suggestion
      Path.type("D:/bar.ex")        #=> :volumerelative
```

Comment (princemaple) [reply lib/elixir/lib/path.ex]: Good eyes.

Comment (josevalim) [reply lib/elixir/lib/path.ex]: `D:bar.ex` is actually valid on Windows. More info here: https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

**Comment by josevalim** [comment]
Thanks for the PR but the path *is* valid. :)

**Comment by princemaple** [comment]
Woah, TIL. Thanks.

**Pull request closed without merging**